src/main/webapp/js/components/fields/field-config-modal.jsx

```diff
@@ -0,0 +1,239 @@
+import React from 'react';
+import Modal from 'react-modal';
+import PropTypes from 'prop-types';
+import { Form } from 'react-final-form';
+import _ from 'lodash';
+
+import ConfirmModal from '../comfirm-modal';
+import renderFormField from '../form/form-utils';
+
+Modal.setAppElement(document.getElementById('root'));
+
+const customStyles = {
+  content: {
+    top: '62%',
+    left: '50%',
+    right: 'auto',
+    bottom: 'auto',
+    marginRight: '-50%',
+    transform: 'translate(-50%, -50%)',
+  },
+  overlay: {
+    overflow: 'auto',
+  },
+};
+
+const FIELD_TYPE_OPTIONS = [
+  { label: '', value: null },
```

Comment:
remove this option, I think it will be better to allow clearing the select (x button in select)